File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int left = 0, right = numbers.length-1;
        
        while(left < right){
            int sum = numbers[left] + numbers[right];
            if ( sum > target){
                right-=1;
            }else if (sum < target) {
                left +=1;
            }else{
                return new int[]{left + 1, right + 1};
            }
        }
        
        return new int[]{};
    }
}

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        
        left = 0
        right = len(numbers) - 1
        
        while True:
     
            sum = numbers[left] + numbers[right]
            
            if sum > target:
                right-=1
            elif sum < target:
                left +=1
            else:
                return [left + 1, right + 1]